README.md

```diff
@@ -237,9 +237,8 @@ profile. Here's an example of overriding some global level default values:
 ```yaml
 profile: sds
 values:
-  global:
-    logging:
-      level: "default:warning" # override from info
+  logging:
+    level: "default:warning" # override from info
```

Comment:
I think this part was correct before?

Reply:
Yes, you're right. Thanks.

Reply:
I mean that Morven is right i.e. there should be no global.